src/middlewares/batchMiddleware.js

```diff
@@ -103,7 +104,6 @@ class Queue {
             return callback(
               apiErrorObject({
                 ...resourceError.error,
-                error_resource_index: resourceErrorIndex,
```

Comment:
no, I need that error_resource_index (but you can rename it `errorResourceIndex`)